CHANGELOG.md

```diff
@@ -20,5 +24,6 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 - Updated image versions from `4.5.20069` to `4.6.20156`.
 
-[Unreleased]: https://github.com/giantswarm/aqua-app/compare/v4.6.0...master
+[Unreleased]: https://github.com/giantswarm/aqua-app/compare/v4.6.1...master
+[v4.6.1]: https://github.com/giantswarm/aqua-app/releases/tag/v4.6.1
```

Comment:
```suggestion
[v4.6.1]: https://github.com/giantswarm/aqua-app/compare/v4.6.0...v4.6.1
```

For consistency with other apps/changelogs, please apply this for previous releases too.